src/pages/equipo.js

```diff
@@ -1,386 +1,413 @@
-                            </Grid>
-                        </Box>
+  const TeamSection = () => {
+    return (
+      <Box className="containerSec">
+        <Box className={clsx("sectionNoPadding")}>
+          <Box className={isDesktop ? "doubleSpacingBox" : ""}>
+            <Box className="spacingBox">
+              <h2>Equipo gerencial</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {ManagementTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
+                    >
+                      <PersonCard
+                        img={useBaseUrl(person.img)}
+                        name={person.name}
+                        position={person.position}
+                        bio={person.bio}
+                        linkedIn={person.linkedIn}
+                        twitter={person.twitter}
+                      />
                     </Box>
-                    <Box className="doubleSpacingBox">
-                        <Box className="spacingBox">
-                            <h2>Mercadeo y ventas</h2>
-                        </Box>
-                        <Box>
-                            <Grid container>
-                                {MarketingSalesTeam.map((person) => (
-                                    <Grid item xs={12} md={6} key={person.name}>
-                                        <Box className="teamPadding" style={{backgroundColor:isMobile? person.color: ""}}>
-                                            <PersonCard 
-                                                img={useBaseUrl(person.img)}
-                                                name={person.name}
-                                                position={person.position}
-                                                bio={person.bio}
-                                                linkedIn={person.linkedIn}
-                                                twitter={person.twitter}
-                                            />
-                                        </Box>
-                                    </Grid>
-                                ))}
-                            </Grid>
-                        </Box>
+                  </Grid>
+                ))}
+              </Grid>
+            </Box>
+          </Box>
+          <Box className="doubleSpacingBox">
+            <Box className="spacingBox">
+              <h2>Mercadeo y ventas</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {MarketingSalesTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
```

Comment:
Please change style={{ backgroundColor: isMobile ? person.color : "" }} to bgColor Box prop